src/actions/complaints.jsx

```diff
@@ -105,7 +105,7 @@ export function getAggregations() {
 export function getComplaints() {
   return ( dispatch, getState ) => {
     const store = getState()
-    const qs = store.query.queryString + '&no_aggs=true'
```

Comment:
Do we need aggregations in the complaint request, or was this just to get the request cycle working? 
Not running aggregations on every request is a bit more efficient. Not huge.